[{"user_id": 1736, "stars": [], "topic_id": 20585, "date_created": 1303333420.3525751, "message": "Line 16 is an else: statement, maybe we are looking at different versions?", "group_id": 81, "id": 761446}, {"user_id": 1736, "stars": [], "topic_id": 20585, "date_created": 1303333648.3302491, "message": "If you use Sublime and want to try the plugin (displays test status in the sublime status bar) it is up at https://github.com/coderanger/sublime-plugins", "group_id": 81, "id": 761503}, {"user_id": 1736, "stars": [], "topic_id": 20585, "date_created": 1303332696.8190451, "message": "No idea about how to integrate it will nose though, I just use normal unittest", "group_id": 81, "id": 761263}, {"user_id": 1806, "stars": [], "topic_id": 20585, "date_created": 1303332827.5103791, "message": "My current work-around just uses 'watch' and notify-send:", "group_id": 81, "id": 761294}, {"user_id": 1806, "stars": [], "topic_id": 20585, "date_created": 1303332841.0113289, "message": "watch -n 10 './manage.py test webservices --settings=settings_test --color || notify-send \"TESTS FAILED\"'", "group_id": 81, "id": 761296}, {"user_id": 1736, "stars": [], "topic_id": 20585, "date_created": 1303333347.787605, "message": "Non-OS X should just bail out since I never finishes the Sublime plugin for them", "group_id": 81, "id": 761431}, {"user_id": 1736, "stars": [], "topic_id": 20585, "date_created": 1303332670.3324759, "message": "PyZen is designed for this", "group_id": 81, "id": 761259}, {"user_id": 1806, "stars": [], "topic_id": 20585, "date_created": 1303333124.30896, "message": "Just tried, but getting a path error from here: https://github.com/coderanger/pyzen/blob/master/pyzen/ui/sublime.py#L14", "group_id": 81, "id": 761360}, {"user_id": 1806, "stars": [], "topic_id": 20585, "date_created": 1303333602.659128, "message": "Looks great so far - thanks for the hard work", "group_id": 81, "id": 761491}, {"user_id": 1736, "stars": [], "topic_id": 20585, "date_created": 1303332893.4702761, "message": "PyZen also integrates with PyNotify for reporting ;-)", "group_id": 81, "id": 761302}, {"user_id": 1736, "stars": [], "topic_id": 20585, "date_created": 1303333297.6211531, "message": "Odd, are you using OS X or Linux? (would have thought Linux from the command line above)", "group_id": 81, "id": 761412}, {"user_id": 1806, "stars": [], "topic_id": 20585, "date_created": 1303333359.675519, "message": "linux - it's just that base_path is uninitialised at line 16.  I fixed it by initialising it to /tmp - now working nicely", "group_id": 81, "id": 761433}, {"user_id": 1806, "stars": [], "topic_id": 20585, "date_created": 1303332542.4934671, "message": "Am using django-nose but want to get a local continuous integration set-up working.  Not found a good solution yet.", "group_id": 81, "id": 761225}, {"user_id": 1736, "stars": [], "topic_id": 20585, "date_created": 1303332678.743727, "message": "And it includes a Django management interface", "group_id": 81, "id": 761261}, {"user_id": 1806, "stars": [], "topic_id": 20585, "date_created": 1303333128.3104761, "message": "no base_path", "group_id": 81, "id": 761361}, {"user_id": 1806, "stars": [], "topic_id": 20585, "date_created": 1303333528.431725, "message": "You're right though - it's a version thing", "group_id": 81, "id": 761468}, {"user_id": 1736, "stars": [], "topic_id": 20585, "date_created": 1303333573.377337, "message": "Yeah, I think I was waiting to be able to test Windows more with the new reloader. Django 1.3 broke the last release since it uses signals now.", "group_id": 81, "id": 761478}, {"user_id": 1736, "stars": [], "topic_id": 20585, "date_created": 1303333506.635437, "message": "Methinks I need to do a new release :-)", "group_id": 81, "id": 761463}, {"user_id": 1806, "stars": [], "topic_id": 20585, "date_created": 1303332589.2361331, "message": "I can't get the runtester option working from django-test-extensions working, plus it doesn't seem to play nicely with django-nose", "group_id": 81, "id": 761234}, {"user_id": 1806, "stars": [], "topic_id": 20585, "date_created": 1303333494.834584, "message": "on my sublime.py, it's \"pid_path = os.path.join(base_path, 'sublime2.pid')\"", "group_id": 81, "id": 761459}, {"user_id": 1806, "stars": [], "topic_id": 20585, "date_created": 1303332615.395108, "message": "Something like nosy looks about right - but it's hardcoded to run nosetests directly - not sure how to hook it up with django", "group_id": 81, "id": 761244}, {"user_id": 1736, "stars": [], "topic_id": 20585, "date_created": 1303332903.0802491, "message": "(or Growl, or the Windows taskbar)", "group_id": 81, "id": 761305}, {"user_id": 1736, "stars": [], "topic_id": 20585, "date_created": 1303333529.612741, "message": "Let me know if you see any other issues though!", "group_id": 81, "id": 761469}, {"user_id": 1736, "stars": [], "topic_id": 20585, "date_created": 1303333776.5766449, "message": "just needs path setting on both ends for Linux, not sure where ST2 stores data on other platforms", "group_id": 81, "id": 761533}, {"user_id": 1806, "stars": [], "topic_id": 20585, "date_created": 1303333213.379519, "message": "Sorry that should be line 16", "group_id": 81, "id": 761389}, {"user_id": 1806, "stars": [], "topic_id": 20585, "date_created": 1303333541.1112001, "message": "The github version is ahead of pip", "group_id": 81, "id": 761470}, {"user_id": 8490, "stars": [], "topic_id": 20585, "date_created": 1303340249.809119, "message": "I run it without notify though. I use something like tdaemon --custom-args='--settings settings_test' --test-program=django", "group_id": 81, "id": 762537}, {"user_id": 8490, "stars": [], "topic_id": 20585, "date_created": 1303340258.379308, "message": "Mostly does the trick.", "group_id": 81, "id": 762538}, {"user_id": 8490, "stars": [], "topic_id": 20585, "date_created": 1303340173.921504, "message": "I've been using tdaemon for a while now. More info here: http://isnomore.net/2010/08/01/automated-python-testing-nose-and-tdaemon/", "group_id": 81, "id": 762525}]